modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc

```diff
@@ -36,28 +35,30 @@ You must configure your Exchange credentials:
   </servers>
 </settings>
 ----
++ 
+NOTE: For information on how to authenticate using a token rather than using credentials, see xref:exchange::to-publish-assets-maven.adoc#publish-and-consume-federated-assets[Publish and Consume Federated Assets].
 . Run `mvn clean deploy` to publish the policy to Exchange.
-
++
 The custom policy is now available for you to apply to APIs that belong to the specified organization.
 
 == Deploying a Policy Created Without Using the Maven Archetype
 
-If you chose not to create your custom policy project using the provided Maven archetypes, you need to manually set up your Exchange deployment configurations:
+If you did not create your custom policy project using the provided Maven archetype, you must manually define the Exchange deployment configuration.
+
+To deploy a policy created without using the Maven archetype:
 
-. In Anypoint Platform, search your organization's ID to use in the following steps.
-* Go to *Access Management > Organization*:
-** Click the name of your organization.
-** Copy the UUID from the browser address. For example, copy _2a4b93c3-7899-4ea7-9374-f787744d8784_ from the URL.
-. Go to your `pom.xml`
-. Replace the groupId with the organization's ID from the previous step.
+. In Anypoint Platform, find the organization ID to use as follows:
+* Go to *Access Management > Organization*.
+* Click the name of your organization.
+* Copy the UUID from the browser address. For example, copy _2a4b93c3-7899-4ea7-9374-f787744d8784_ from the URL.
+. Update your `pom.xml` file as follows: 
+* Replace the `groupId` value with the organization ID from the preceding step:
 [source,XML,linenums]
 +
 --
 <groupId>{orgId}</groupId>
 --
-. Add the following properties replacing the {orgId} for the organization's id from the previous steps:
-
-
+* Add the following properties and replace `{orgId}` with the organization ID from the preceding step:
```

Comment:
It would be a better use of "following" and the colon to reorder this:
```suggestion
* Replace `{orgId}` with the organization ID from the preceding step and add the following properties:
```

Reply:
This one is a little tough because they actually have to add these lines to the file. They have to add the properties first.